content/ko/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -0,0 +1,399 @@
+---
+title: 노드에 파드 할당하기
+content_template: templates/concept
+weight: 30
+---
+
+
+{{% capture overview %}}
+
+{{< glossary_tooltip text="파드" term_id="pod" >}}를 특정한 {{< glossary_tooltip text="노드(emf)" term_id="node" >}}에서만 동작하도록 하거나,
+특정 노드들을 선호하도록 제한할 수 있다.
+이를 수행하는 방법에는 여러 가지가 있으며, 권장되는 접근 방식은 모두
+[레이블 셀렉터(label selector)](/ko/docs/concepts/overview/working-with-objects/labels/)를 사용하여 선택한다.
+보통 스케줄러가 자동으로 합리적인 배치를 수행하기에 이런 제약 조건은 필요하지 않지만
+(예: 노드들에 걸쳐 파드를 분배하거나, 여유 자원이 부족한 노드에 파드를 배치하는 등)
+간혹 파드가 착륙하는 노드에 대해 더 많은 제어를 원할 수 있는 상황이 있다.
+예를 들어 SSD가 장착된 머신에 파드가 연결되도록 하거나 또는 동일한 가용성 영역(availability zone)에서
+많은 것을 통신하는 두 개의 서로 다른 서비스의 파드를 같이 배치할 수 있다.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 노드 셀렉터(nodeSelector)
+
+`nodeSelector` 는 가장 간단하고 권장하는 노드 선택 제약 조건의 형태이다.
+`nodeSelector` 는 PodSpec의 필드이다. 이는 키-값 쌍의 매핑으로 지정한다. 파드가 노드에서 동작할 수 있으려면,
+노드는 키-값의 쌍으로 표시되는 레이블을 각자 가지고 있어야 한다(이는 추가 레이블을 가지고 있을 수 있다).
+대부분의 일반적인 사용은 하나의 키-값 쌍이다.
+
+`nodeSelector` 를 어떻게 사용하는지 예시를 통해 알아보도록 하자.
+
+### 0 단계: 사전 준비
+
+이 예시는 쿠버네티스 파드에에 대한 기본적인 이해를 하고 있고 [쿠버네티스 클러스터가 설정](/ko/docs/setup/)되어 있다고 가정한다.
+
+### 1 단계: 노드에 레이블 붙이기
+
+`kubectl get nodes` 를 실행해서 클러스터 노드 이름을 가져온다. 이 중에 레이블을 추가하기 원하는 것 하나를 선택한 다음에 `kubectl label nodes <노드 이름> <레이블 키>=<레이블 값>` 을 실행해서 선택한 노드에 레이블을 추가한다. 예를 들어 노드의 이름이 'kubernetes-foo-node-1.c.a-robinson.internal' 이고, 원하는 레이블이 'disktype=ssd' 라면, `kubectl label nodes ubernetes-foo-node-1.c.a-robinson.internal disktype=ssd` 를 실행한다.

```

Comment:
nit:
```suggestion
`kubectl get nodes` 를 실행해서 클러스터 노드 이름을 가져온다. 이 중에 레이블을 추가하기 원하는 것 하나를 선택한 다음에 `kubectl label nodes <노드 이름> <레이블 키>=<레이블 값>` 을 실행해서 선택한 노드에 레이블을 추가한다. 예를 들어 노드의 이름이 'kubernetes-foo-node-1.c.a-robinson.internal' 이고, 원하는 레이블이 'disktype=ssd' 라면, `kubectl label nodes kubernetes-foo-node-1.c.a-robinson.internal disktype=ssd` 를 실행한다.
```

Reply:
@KimMJ Please update PR according to this @sftim 's comment.